boards/nrf5340dk-app/include/periph_conf.h

```diff
@@ -112,6 +110,37 @@ static const pwm_conf_t pwm_config[] = {
 #define PWM_NUMOF           ARRAY_SIZE(pwm_config)
 /** @} */
 
+/**
+ * @name    SPI configuration
+ * @{
+ */
+static const spi_conf_t spi_config[] = {
+    {
+        .dev  = NRF_SPIM1_S,
+        .sclk = GPIO_PIN(0, 17),
+        .mosi = GPIO_PIN(0, 13),
+        .miso = GPIO_PIN(0, 14),
+    }
+};
+
+#define SPI_NUMOF           ARRAY_SIZE(spi_config)
+/** @} */
+
+/**
+ *  * @name    I2C configuration
+ *   * @{
+ *    */
+static const i2c_conf_t i2c_config[] = {
+    {
+	    .dev = NRF_TWIM2_S,
+		.scl = GPIO_PIN(1, 3),
```

Comment:
```suggestion
        .dev = NRF_TWIM2_S,
        .scl = GPIO_PIN(1, 3),
```
doccheck doesn't like tabs

Reply:
Sorry...
That's what happen when you don't use your usual text editor :(
Fixed.